### PR TITLE
Fixes air shoes implant exploit using boost while hardcrit or dead

### PR DESCRIPTION
**Pull request by ynot01**
# Document the changes in your pull request

Can no longer use it in hardcrit
Can still use it in softcrit bc im nice

# Changelog

:cl:  
bugfix: Can no longer use air shoes jump boost in hardcrit or death
/:cl:

### CODE REVIEW

**Comment by Moltijoe** [comment]
Hold up, isnt that what the ability check flags do?

Edit: Oh, probably because im overwriting the original isavailable proc.
In that case, the other abilities probably dont respect check flags either

**Comment by Moltijoe** [comment]
> Edit: Oh, probably because im overwriting the original isavailable proc. 

Exactly what happened

File: code/modules/surgery/organs/augment_legs.dm [hunk 1]
```diff
@@ -214,7 +214,7 @@
 	holder = user
 
 /datum/action/innate/boost/IsAvailable()
-	if(COOLDOWN_FINISHED(src, dash_cooldown))
```
Comment (Moltijoe) [inline code/modules/surgery/organs/augment_legs.dm:217]: just add 
`. = ..()`
above this line instead

**Review by Moltijoe — CHANGES_REQUESTED** [review]

**Review by Moltijoe — APPROVED** [review]

**Comment by ynot01** [comment]
sir if you write . = ..() and then return a value after that it's going to return the value not ..()

**Pull request merged**